docs/advanced-guides/cloud-in-a-box.md

```diff
@@ -45,8 +45,14 @@ There are two types of Cloud in a Box.
 
 ## Installation
 
-1. Download one of the Cloud in a Box images of type sandbox
+:::warning
+The images currently download and install the 
+[latest release of the installation-scripts](https://github.com/osism/cloud-in-a-box),
+therefore it is mandatory to update the installation media when the base release changes.
+The installation of historical releases is currently not supported. 
+:::
 
+1. Download one of the Cloud in a Box images of type sandbox</br>
```

Comment:
Is there not better way than using HTML? I would prefer to have Markdown only.

Reply:
Removed that